x-pack/legacy/plugins/ml/public/services/ml_api_service/data_frame.js

```diff
@@ -53,15 +53,15 @@ export const dataFrame = {
       data: obj
     });
   },
-  startDataFrameTransformsJob(jobId) {
+  startDataFrameTransformsJob(jobId, force = false) {
```

Comment:
Could this just be called `startDataFrameTransform` / `stopDataFrameTransform` ?